tests/prepare_input_files_lists/prepare_input_files_lists_schannel.py

```diff
@@ -0,0 +1,57 @@
+import os
```

Comment:
Do we really need to make one script per analysis and have to copy duplicate the code everywhere? It would much better to extract the common piece of code somewhere rather than to have to maintain N duplication of the same thing...